lib/models/application.js

```diff
@@ -1562,6 +1565,141 @@ const getApplicationModel = function (deps, opts) {
 			 */
 			remove: envVarModel.remove,
 		},
+
+		/**
+		 * @namespace balena.models.application.invite
+		 * @memberof balena.models.application
+		 */
+		invite: {
+			/**
+			 * @summary Get a single invite
+			 * @name get
+			 * @public
+			 * @function
+			 * @memberof balena.models.application.invite
+			 *
+			 * @param {Number} id - invite id
+			 * @param {Object} [options={}] - extra pine options to use
+			 * @fulfil {Object} - invite
+			 * @returns {Promise}
+			 *
+			 * @example
+			 * balena.models.application.invite.get(51).then(function(invite) {
+			 * 	console.log(invite);
+			 * });
+			 */
+			get: inviteModel.get,
+
+			/**
+			 * @summary Get all invites
+			 * @name getAll
+			 * @public
+			 * @function
+			 * @memberof balena.models.application.invite
+			 *
+			 * @description
+			 * This method returns all invites.
+			 *
+			 * @param {Object} [options={}] - extra pine options to use
+			 * @fulfil {Object[]} - invites
+			 * @returns {Promise}
+			 *
+			 * @example
+			 * balena.models.application.invite.getAll().then(function(invites) {
+			 * 	console.log(invites);
+			 * });
+			 */
+			getAll: inviteModel.getAll,
```

Comment:
How about all these jsdoc comments to the `application-invite.ts` and just import and assign it here in the `invite` prop (instead of assigning each prop separately)?